src/AbstractFuzzyQuery.php

```diff
@@ -68,7 +75,10 @@ public function __toString()
             $raw = $this->exclusion . $raw;
         } else {
             if (!empty($this->fuzzyness)) {
-                $raw .= '~' . $this->fuzzyness;
+                $raw .= '~';
+                if (is_integer($raw)) {
```

Comment:
Since test upper uses `is_numeric()` you should use `is_numeric()` here too, or even better test against `self::FUZZY_AUTO` and completely drop `is_numeric()` call.